docs/edgeanalytics/kafka-options.md

```diff
@@ -24,7 +24,7 @@ The rest of this document will cover how to install and deploy several flavors o
 
 ## Using an existing Kafka deployment
 
-If a user already has a Kafka environment ready, use that environment by [configuring](https://ibmstreams.github.io/streamsx.kafka/docs/user/overview/) their `KafkaConsumer` and `KafkaProducer` operators and their property file for their Streams application to access the existing Kafka environment.
+If you already has a Kafka environment ready, use that environment by [configuring](https://ibmstreams.github.io/streamsx.kafka/docs/user/overview/) your `KafkaConsumer` and `KafkaProducer` operators and your property file for the Streams application to access the existing Kafka environment.
```

Comment:
Typo - if you already _have_ a Kafka environment.
The Kafka toolkit page needs some work, I'd link to this getting started doc instead: http://ibmstreams.github.io/streamsx.documentation/docs/messaging/kafka-operators-getting-started/#connecting-to-ibm-event-streams